common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -514,7 +514,10 @@ def test_export_to_xml(self):
            <transcript language="ge" src="german_translation.srt" />
            <transcript language="ua" src="ukrainian_translation.srt" />
          </video>
-        ''')
+        """
+        # whitespace screws up etree comparison
+        expected = ''.join(line.strip() for line in expected.split('\n'))
+        expected = etree.fromstring(expected)
         self.assertXmlEqual(expected, xml)
```

Comment:
Perhaps as cleaner way to do this would be to make assertXmlEqual optionally strip whitespace from the text/tail? Or, alternately, you could use an lxml parser that strips whitespace (https://stackoverflow.com/questions/3310614/remove-whitespaces-in-xml-string)